mne/utils.py

```diff
@@ -510,6 +510,21 @@ def _reject_data_segments(data, reject, flat, decim, info, tstep):
     return data, drop_inds
 
 
+def _get_inst_data(inst):
```

Comment:
also could add a `copy=False` flag along with `deepcopy` at the end in case somebody would want to copy the data before returning it.